Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+ToolbarDelegate.swift

```diff
@@ -539,6 +540,9 @@ extension BrowserViewController: TopToolbarDelegate {
         guard let self else { return }
         
         if finalizedRecognition.status {
+          // Feedback indicating recognition is finalized
+          AudioServicesPlayAlertSound(SystemSoundID(kSystemSoundID_Vibrate))
+          UIImpactFeedbackGenerator(style: .medium).bzzt()
```

Comment:
Why do we need both calls you added? I thought the bzzt one would be enough

Reply:
One of them audio sound indicator, the other one is actual vibration. If phone vibration is off, there will not be a feedback for that device. AudioServicesPlayAlertSound is also used in Sync QR code scan.